web/src/beta/features/Editor/Map/LayerStylePanel/Editor/StyleInterface/convert.ts

```
@@ -0,0 +1,140 @@
+  StyleCondition,
+  StyleNode,
+  StyleNodes,
+  StyleValue
+} from "./types";
+
+export const convertToStyleNodes = (
+  layerStyle: LayerStyle | undefined
+): StyleNodes => {
+  return appearanceTypes.reduce((acc, cur) => {
+    return {
+      ...acc,
```

Biome

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)